### ISSUE TITLE
predict function error

### ISSUE THREAD

**Issue by hamdanihamza** [cyprienruffino/CTCModel#12]
When I use the prediction function it returns empty lists
```
Prediction : []  -- Label :  [3. 3. 1. 4.]
Prediction : []  -- Label :  [1. 1.]
Prediction : []  -- Label :  [0. 7. 2.]
Prediction : []  -- Label :  [6. 4.]
Prediction : []  -- Label :  [5. 8.]
Prediction : []  -- Label :  [0. 6. 9.]
Prediction : []  -- Label :  [6. 8. 6. 6.]
Prediction : []  -- Label :  [4. 1.]
Prediction : []  -- Label :  [8. 4.]
Prediction : []  -- Label :  [2. 3. 3. 1.]
```
I Train the model on at least 10 eopchs. I am actually working on audio data but I encounter the same issue with the data provided in the example !

**Comment by JohnLoveMm** [comment]
probably i think,your training loss did not descent, the loss around 10, i think your network will be working.that just a advice.

**Comment by JohnLoveMm** [comment]
> When I use the prediction function it returns empty lists
> 
> ```
> Prediction : []  -- Label :  [3. 3. 1. 4.]
> Prediction : []  -- Label :  [1. 1.]
> Prediction : []  -- Label :  [0. 7. 2.]
> Prediction : []  -- Label :  [6. 4.]
> Prediction : []  -- Label :  [5. 8.]
> Prediction : []  -- Label :  [0. 6. 9.]
> Prediction : []  -- Label :  [6. 8. 6. 6.]
> Prediction : []  -- Label :  [4. 1.]
> Prediction : []  -- Label :  [8. 4.]
> Prediction : []  -- Label :  [2. 3. 3. 1.]
> ```
> 
> I Train the model on at least 10 eopchs. I am actually working on audio data but I encounter the same issue with the data provided in the example !

probably i think,your training loss did not descent, the loss around 10, i think your network will be working.that just a advice.

**Comment by cyprienruffino** [comment]
That is a weird property that some networks trained with the CTC loss have, that they tend to plateau a bit at the start, predicting only blank labels. After a bunch epochs, the loss will start to steadily decrease and the network will start making predictions.
